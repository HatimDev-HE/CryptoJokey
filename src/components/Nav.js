/* eslint-disable jsx-a11y/label-has-associated-control */
/* eslint-disable import/prefer-default-export */
import React from 'react';
import { NavLink } from 'react-router-dom';
import mic from './Assets/mic.svg';
import Back from './Assets/caret-left.svg';
import gear from './Assets/gear.svg';

function Nav() {
  return (
    <>
      <div className="topnav">
        <li className="navHome">
          <NavLink to="/">
            <img src={Back} alt="" height={30} width={30} />
          </NavLink>
        </li>
        <li className="topHome">JokeR</li>
        <li><img src={mic} alt="" width={30} height={30} /></li>
        <li><img src={gear} alt="" width={30} height={30} /></li>
      </div>

      <div className="headall">
        <header>Welcome to CryptoJokey</header>
        <p className="slogan">&lsquo;In crypto we roll&lsquo;</p>
      </div>

    </>
  );
}

export default Nav;
