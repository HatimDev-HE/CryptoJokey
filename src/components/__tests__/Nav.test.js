import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../../redux/configstore';
import Nav from '../Nav';

describe('testing Nav comp', () => {
  test('Nav to match snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <Nav />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
