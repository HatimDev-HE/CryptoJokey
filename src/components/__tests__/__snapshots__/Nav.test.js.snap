// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Nav comp Nav to match snapshot 1`] = `
Array [
  <div
    className="topnav"
  >
    <li
      className="navHome"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <img
          alt=""
          height={30}
          src="caret-left.svg"
          width={30}
        />
      </a>
    </li>
    <li
      className="topHome"
    >
      JokeR
    </li>
    <li>
      <img
        alt=""
        height={30}
        src="mic.svg"
        width={30}
      />
    </li>
    <li>
      <img
        alt=""
        height={30}
        src="gear.svg"
        width={30}
      />
    </li>
  </div>,
  <div
    className="headall"
  >
    <header>
      Welcome to CryptoJokey
    </header>
    <p
      className="slogan"
    >
      ‘In crypto we roll‘
    </p>
  </div>,
]
`;
